registry/application/services/service_publisher_service.py

```diff
@@ -188,25 +188,28 @@ def notify_approval_team_when_user_submit_for_approval(slack_msg):
         utils.send_slack_notification(slack_msg=slack_msg, slack_url=slack_url, slack_channel=SLACK_CHANNEL_FOR_APPROVAL_TEAM)
 
     def submit_service_for_approval(self, payload):
+        # we should get org_id from organization service for given org_uuid than from payload
+        org_id = payload["org_id"]
```

Comment:
Code is explaining comments, do we really need comments?

Reply:
When code becomes production-ready, we need limited comments, methods are self-explanatory.
But when you are developing, I don't see any problem with this.
def submit_for_approval():
     # do 1
     implementation 
    # do 2
    implementation
   # do 3
   implementation